### PR TITLE
(fix|COS-2812): Contract UI issues part 2

### PR DESCRIPTION
**Pull request by Kasia9090**
- fix datepicker
- add go to billing details button on invoice preview
- fix date selection

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

## Summary by CodeRabbit

- **New Features**
	- Added conditional rendering and blur effects in various invoice components based on billing settings.
	- Updated the user interface for editing billing details in invoices.

- **Bug Fixes**
	- Adjusted text styles in the renewal confirmation dialog for better readability.

- **Style**
	- Updated logo and style adjustments in invoice components for enhanced visual presentation.

- **Refactor**
	- Streamlined billing details handling across components for improved performance and user experience.

### CODE REVIEW

File: packages/apps/spaces/ui/form/DatePicker/DatePickerUnderline.tsx [hunk 1]
```diff
@@ -92,6 +90,7 @@ export const DatePickerUnderline: React.FC<DatePickerProps> = ({
             DateTimeUtils.abreviatedMonth,
           )
         }
+        value={value as Date}
```
Comment (acalinica) [inline packages/apps/spaces/ui/form/DatePicker/DatePickerUnderline.tsx:93]: I see that this one is done with Chakra. I know that Mihai and Steyn worked on a new aligned DatePicker or Calendar in `DatePicker2.tsx` which is done with tw. Would be worth migrating to that one as we'll pretty soon remove chakra from the codebase.

Comment (Kasia9090) [reply packages/apps/spaces/ui/form/DatePicker/DatePickerUnderline.tsx:93]: it's not chakra, it's react-date-picker. I will remove the Box as the last part of chakra remaining

Comment (acalinica) [reply packages/apps/spaces/ui/form/DatePicker/DatePickerUnderline.tsx:93]: ok but we still plan on removing every other DatePicker and using the one just implemented.

Comment (acalinica) [reply packages/apps/spaces/ui/form/DatePicker/DatePickerUnderline.tsx:93]: Ok, let's get this through and circle back later when we approach removing chakra.

**Review by acalinica — APPROVED** [review]

**Pull request merged**